vmdb/app/models/miq_schedule.rb

```diff
@@ -334,8 +334,8 @@ def depot_hash
   end
 
   def next_interval_time
-    unless self.valid? || self.errors.on(:run_at).empty?
-      $log.warn("Miq(Schedule.next_interval_time) Invalid schedule [#{self.id}] [#{self.name}]: #{self.errors.on(:run_at).to_miq_a.join(", ")}")
+    unless self.valid? || self.errors.get(:run_at).nil? || self.errors.get(:run_at).empty?
```

Comment:
Do we need to check for nil here?  Im hoping rails returns an empty array always.
Even if we do need to check for nil, we can use .blank? to check for nil and empty.

Am I the only one that finds this hard to read?

Maybe I'm weird but I find this easier to read:
```ruby
if !valid? && !errors.get(:run_at).blank?
```